sql/flow.go

```diff
@@ -26,21 +23,62 @@ const (
 )
 
 var (
-	DockerClientInit = NewDockerClient
-	ioCopy           = io.Copy
+	DockerClientInit   = NewDockerClient
+	IoCopy             = io.Copy
+	PrintBuildingSteps = printBuildingSteps
+	Println            = fmt.Println
 )
 
 func getContext(filePath string) io.Reader {
 	ctx, _ := archive.TarWithOptions(filePath, &archive.TarOptions{})
 	return ctx
 }
 
+func printBuildingSteps(r io.Reader) error {
```

Comment:
```suggestion
	Println            = fmt.Println
)

func getContext(filePath string) io.Reader {
	ctx, _ := archive.TarWithOptions(filePath, &archive.TarOptions{})
	return ctx
}

var PrintBuildingSteps = func(r io.Reader) error {
```
nit: a simpler way to do the same thing

Reply:
Yes, but I think it looks cleaner having it as separate function - not as inline function in vars.